.codeclimate.yml

```diff
@@ -11,9 +11,12 @@ engines:
         javascript:
   eslint:
     enabled: true
+    config:
+      config: interface/eslint.conf.js
```

Comment:
Is this the right `.yml` syntax here or a typo? Seems odd to have config twice like that.